runtime/drivers/duckdb/olap.go

```diff
@@ -221,20 +221,33 @@ func (c *connection) AlterTableColumn(ctx context.Context, tableName, columnName
 
 // CreateTableAsSelect implements drivers.OLAPStore.
 // We add a \n at the end of the any user query to ensure any comment at the end of model doesn't make the query incomplete.
-func (c *connection) CreateTableAsSelect(ctx context.Context, name string, view bool, sql string, tableOpts map[string]any) error {
+func (c *connection) CreateTableAsSelect(ctx context.Context, name string, view bool, sql, beforeCreateSQL, afterCreateSQL string, tableOpts map[string]any) error {
```

Comment:
Could we pass the input props in the `CreateTableAsSelect` and `InsertTableAsSelect` functions instead of passing the different options separately?

Reply:
Done